### PR TITLE
Use custom nhost/postgres image with correct roles

### PR DESCRIPTION
**Pull request by nunopato**
## Description
Use custom postgres image `nhost/postgres:14.5` based on `postgres:14.5`

## Problem
With the work done to move away from AWS RDS, we are using a custom postgres image with specific roles for all services.

## Solution
Start using the custom postgres image and pass in the right roles to all services

## Notes
This version should only be used by projects already migrated from RDS.

### CODE REVIEW

File: nhost/compose/config.go [hunk 1]
```diff
@@ -171,6 +171,27 @@ func (c Config) postgresConnectionString() string {
 	return fmt.Sprintf("postgres://%s:%s@%s:%d/%s", user, password, SvcPostgres, dbPort, db)
 }
 
+func (c Config) hasuraConnectionString() string {
```
Comment (elephant3) [inline nhost/compose/config.go:174]: Why is this called a hasura connection string and not postgres?

Comment (elephant3) [reply nhost/compose/config.go:174]: You could just change the code in `postgresConnectionString()`

Comment (elephant3) [inline nhost/compose/config.go:174]: I see 3 identical functions that have only different usernames. Make a single one which takes an a single `username`argument

```
func (c Config) postgresConnectionStringWithUsername(username string) string {
	postgresEnv := c.postgresServiceEnvs()
	db := postgresEnv[envPostgresDb]

	return fmt.Sprintf("postgres://%s@%s:%d/%s", username,  SvcPostgres, dbPort, db)
}
```

File: nhost/compose/wrapper.go [hunk 2]
```diff
@@ -70,6 +72,18 @@ func (w Wrapper) ensureFoldersExistForDockerVolumes(workdir, gitBranch string) e
 		}
 	}
 
+	// write pg_hba_local.conf
+	hbaFile := filepath.Join(dotNhostFolder, DbDataDirGitBranchScopedPath(gitBranch, "pg_hba_local.conf"))
+	hbaFileContent := `
+  local all all trust
+	host all all all trust
```
Comment (elephant3) [inline nhost/compose/wrapper.go:79]: image

let's use either spaces or tabs, but not a mix

File: nhost/compose/config.go (hunk 1 above)
Comment (nunopato) [reply nhost/compose/config.go:174]: Because this is the connection string we pass to hasura. I am changing it to 1 function only.

**Review by elephant3 — APPROVED** [review]

**Pull request merged**